types/mongodb/index.d.ts

```diff
@@ -1812,7 +2565,7 @@ type AcceptedFields<TSchema, FieldType, AssignableType> = {
     readonly [key in KeysOfAType<TSchema, FieldType>]?: AssignableType;
 };
 
-/** It avoid uses fields of non Type */
+/** It avoids using fields with not acceptable types */
```

Comment:
This sentence didn't make any sense to me so I tried to fix it but I'm not entirely sure if what I added is what the original author meant.